src/code/reducers/helpers/load-state-from-authoring.js

```diff
@@ -105,10 +105,24 @@ export function loadStateFromAuthoring(state, authoring, progress={}) {
         template = templateName && templates[templateName];
   if (!template) return state;
 
+  let userChangeableGenes = authoredChallenge.userChangeableGenes;
```

Comment:
OK, this is a good point - it's needlessly restrictive to force the names of the fields like this. I agree with leaving it for now, but when we do end up addressing `visibleGenes` (where a change like this is more likely to get used), it should be worth updating `userChangeableGenes` as well.